src/lib/components/progressBar/progressBar.module.scss

```diff
@@ -15,7 +15,7 @@
   font-size: 12px;
```

Comment:
Shouldn't this be in a themify block too?

Reply:
yes! fixed!